composer.json

```diff
@@ -23,5 +23,14 @@
     "psr-4": {
       "Linio\\DynamicFormBundle\\": "tests/"
     }
+  },
+  "scripts": {
```

Comment:
@csanjuanc I recommend you to change the file github actions to use composer scripts: https://github.com/LinioIT/dynamic-form-bundle/blob/master/.github/workflows/php.yml#L31

Something like:
```
 run: composer phpunit
```

Reply:
sure!